src/emit/substrate/mod.rs

```diff
@@ -1851,3 +1853,47 @@ fn event_id<'b>(
 
     Some(binary.context.i8_type().const_int(event_id as u64, false))
 }
+
+/// Print the return code of API calls to the debug buffer.
+fn log_return_code<'b>(binary: &Binary<'b>, api: &'static str, code: &IntValue) {
```

Comment:
`IntValue` is just a wrapper for an LLVM pointer: `LLVMValueRef`, you can pass it by value. Now you're passing a pointer to a pointer.